agent/exploits/webexploit.py

```diff
@@ -86,14 +90,24 @@ def check(self, target: definitions.Target) -> list[definitions.Vulnerability]:
             return vulnerabilities
 
         for pattern in self.match_pattern:
-            if pattern.search(resp.text) is not None:
+            if (
+                pattern.search(resp.text) is not None
+                or pattern.search(str(resp.headers)) is not None
+            ):
```

Comment:
This is duplicate move it to a private method.

Reply:
I've only changed to make the detection better , and having it there seems more clear .